test_get_token, test_register_guest, test_register_item は問題なく動作しているようです．
一方，以下のテストが成功しません．
この件は，今週末にでも相談させてください．

```
(ct_prototype) Silver ct_prototype2/tests$ python test_client.py
test line id is qvm8k
...FFF
======================================================================
FAIL: test_retrieve_guest (__main__.TestClient)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_client.py", line 55, in test_retrieve_guest
    self.assertEqual(response.status_code, 200)
AssertionError: 404 != 200

======================================================================
FAIL: test_retrieve_guest_items (__main__.TestClient)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_client.py", line 110, in test_retrieve_guest_items
    self.assertEqual(response.status_code, 200)
AssertionError: 500 != 200

======================================================================
FAIL: test_retrieve_items (__main__.TestClient)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_client.py", line 84, in test_retrieve_items
    self.assertEqual(response.status_code, 200)
AssertionError: 500 != 200

----------------------------------------------------------------------
Ran 6 tests in 10.225s

FAILED (failures=3)
```

--- a/client.py
+++ b/client.py
@@ -109,7 +109,7 @@ class Client:
         response = requests.post(url, data, headers = headers)
         return response
 
-    def register_item(self, parmas):
+    def register_item(self, params):
         """
         CALL /api/items
         """
@@ -183,7 +183,7 @@ class Client:
         response = requests.post(url, data, headers = headers)
         return response
 
-    def retrieve_guest_items(self, parmas):
+    def retrieve_guest_items(self, params):
         """
         CALL /api/guest-searchitems
         """

